admin_guide/managing_networking.adoc

```diff
@@ -1147,3 +1147,50 @@ objects:
       - podSelector: {}
 ...
 ----
+
+[[admin-guide-enabling-hsts]]
+== Enabling HTTP Strict Transport Security
+
+HTTP Strict Transport Security (HSTS) policy is a security enhancement, which
+ensures traffic to a host will always be HTTPS. This is useful for ensuring
+secure interactions with websites, or to offer secure application for the user's
+benefit.
+
+HSTS adds a Strict Transport Security header to HTTPS responses from the site.
+The client (often a browser) reads the header and from then on enforces the
+policy. All requests to the HTTP URL are changed to HTTPS before the request is
+sent, eliminating the need for a redirect.
+
+When not enabled, any HTTP requests to a host that require HTTPS instead
```

Comment:
This redirect needs to be set up in the route. Its not automatic. By default the request is just dropped. So for this to work reasonably the redirect is set up so that the first time a client requests the target it get a redirect to HTTPS. on the next try, using HTTPS, the client gets the https header in the response. From then on it knows to just send HTTPS.